project/src/com/google/daggerquery/executor/models/Query.java

```diff
@@ -153,7 +149,7 @@ public Query(String typeName, String... parameters) {
       }
     }
 
-    throw new NotImplementedException();
+    throw new UnsupportedOperationException("Query with specified name is not supported yet.");
```

Comment:
Would you mind including the query name in this string?

Reply:
Yep, I have added it ⭐️